contents.md

```diff
@@ -1,76 +0,0 @@
-# Summary
```

Comment:
This file would be required for the gitbooks stuff, but could be slimmed down a lot

Reply:
Rebuilt (and renamed back to the default name) the summary.md